src/components/LoadingIndicator/LoadingIndicator.tsx

```diff
@@ -18,17 +17,11 @@ export type Props = {
 
   /**
    * Aria label of the oval. Default is "loading". Will be overridden if ariaLabel is passed in props
-   * (Shadowed from OvalProps)
-   *
-   * See: https://mhnpd.github.io/react-loader-spinner/docs/components/oval
    */
   ariaLabel?: string;
 
   /**
    * Whether the oval is visible. Default is true.
-   * (Shadowed from OvalProps)
-   *
-   * See: https://mhnpd.github.io/react-loader-spinner/docs/components/oval
    */
   visible?: boolean;
```

Comment:
What's the point of having an invisible loading spinner?

Reply:
`visible` for loaders allows for those cases where you want to preserve content when the loading indicator would otherwise be added and removed from the UI in quick succession. We can use this prop to handle transitions or other behaviors